{{ cookiecutter.repo_name }}/docs/contributor_guide/writing_sphinx_documentation.md

```diff
@@ -58,15 +58,6 @@ create a ReST file, and add the following line to reference the `{{ cookiecutter
 Then, elsewhere in the body, [call the `autosummary` directive to generate the
 docstrings as ReST stub files][sphinx-autosummary].
 
-```rest
```

Comment:
The removal of these requires a quick discussion.

Their purpose is to demonstrate how to use autosummary, but without a demo .py and python functions this is not possible. Ideally I would include the code as a comment (therefore it won't run and cause an error) to still demonstrate to the user how to use them, but I can not work out how to do that.